diff -Naru testdir-orig/test_listlbr_utf8.in testdir/test_listlbr_utf8.in
--- a/src/testdir/test_listlbr_utf8.in	2015-01-21 11:21:22.000000000 +0000
+++ b/src/testdir/test_listlbr_utf8.in	2015-01-22 07:57:25.000000000 +0000
@@ -2,7 +2,7 @@
 
 STARTTEST
 :so small.vim
-:if !exists("+linebreak") | e! test.ok | w! test.out | qa! | endif
+:if !exists("+linebreak") || !has("conceal") | e! test.ok | w! test.out | qa! | endif
 :so mbyte.vim
 :if &enc !=? 'utf-8'|:e! test.ok|:w! test.out|qa!|endif
 :10new|:vsp|:vert resize 20
